src/games/strategy/engine/framework/startup/ui/PBEMSetupPanel.java

```diff
@@ -204,6 +205,7 @@ private void loadForumPosters(final GameData data) {
     forumPosters.add((IForumPoster) findCachedOrCreateNew(NullForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(AxisAndAlliesForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(TripleAWarClubForumPoster.class));
```

Comment:
Okay. Removing warclub PBF probably should have been first before we added more options. Let's be sure to do the removal soon (if not now - we're already testing the PBF feature, we avoid the duplicated testing if we lump it together.)